### PR TITLE
Feature/alp 182/asset search

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

**Review by FIZLIN — APPROVED** [review]
LGTM

File: packages/app/src/scenes/odysseyCreator/stores/SpawnAssetStore/SpawnAssetStore.ts [hunk 1]
```diff
@@ -118,6 +120,7 @@ const SpawnAssetStore = types
         {
           parent_id: worldId,
           space_name: self.navigationObjectName,
+          // TODO: What is it for? Discussion !!!
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/odysseyCreator/stores/SpawnAssetStore/SpawnAssetStore.ts:123]: It's magic id of "Customely placed objects" object/space type.

File: packages/app/src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx [hunk 2]
```diff
@@ -189,9 +189,10 @@ const MoleculesPage: FC = () => {
         <styled.Name>
           <Heading label="Component «SearchInput»" type="h2" align="left" />
         </styled.Name>
-        <styled.Section>
-          <SearchInput placeholder="Search for..." onChange={(value) => {}} delay={300} />
-        </styled.Section>
+        <styled.SectionGrid2>
+          <SearchInput placeholder="Search for..." onChange={(value) => {}} />
+          <SearchInput variant="secondary" placeholder="Search for..." onChange={(value) => {}} />
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx:194]: Hmm, actually 2 inputs?

**Review by dmitry-yudakov — APPROVED** [review]
It seems we have 2 inputs in one place but the rest is good.

File: packages/app/src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx (hunk 2 above)
Comment (PAKovalev) [reply packages/app/src/scenes/storyBook/pages/MoleculesPage/MoleculesPage.tsx:194]: It is a storybook.

**Pull request merged**